examples/service-clients/azure-client/external-controller/tests/index.ts

```diff
@@ -154,26 +146,34 @@ export async function createContainerAndRenderInElement(
 /**
  * For local testing we have two div's that we are rendering into independently.
  */
-async function setup() {
+async function setup(): Promise<void> {
+	// Since this is a single page Fluid application we are generating a new document id
+	// if one was not provided
+	const createNew = window.location.hash.length === 0;
+	if (createNew) {
+		window.location.hash = Date.now().toString();
+	}
+	const documentId = window.location.hash.substring(1);
```

Comment:
There is some contention regarding "document ID" vs "container ID" terminology. I believe the current guidance is to use `container ID`. @ChumpChief to confirm.

Reply:
Sorry I missed this - yes, please use "container" rather than "document" in all cases.